core/modules/map.rs

```diff
@@ -1996,100 +1997,118 @@ pub fn script_origin<'a>(
 }
 
 fn render_js_wasm_module(specifier: &str, wasm_deps: WasmDeps) -> String {
-  // NOTE(bartlomieju): it's unlikely the generated file will have more lines,
-  // but it's better to overallocate than to have to mem copy.
-  let mut src = Vec::with_capacity(512);
+  struct ImportInfo {
+    key_escaped: String,
+    escaped_named_imports: Vec<String>,
+  }
 
-  fn aggregate_wasm_module_imports(
-    imports: &[wasm_dep_analyzer::Import],
-  ) -> IndexMap<String, Vec<String>> {
-    let mut imports_map = IndexMap::default();
+  fn aggregate_wasm_module_imports<'a>(
+    imports: &'a [wasm_dep_analyzer::Import],
+  ) -> IndexMap<&'a str, ImportInfo> {
+    let mut imports_map = IndexMap::with_capacity(imports.len());
 
     for import in imports.iter().filter(|i| {
       matches!(i.import_type, wasm_dep_analyzer::ImportType::Function(..))
```

Comment:
Btw, this seems wrong. Shouldn't we be importing all imports?

Reply:
Good point, I haven't seen a Wasm module though that had something else an import. But we definitely should support it. Let's track in a new issue though.